### DISCUSSION TITLE
블로그 등록 어디에서 해야 하나요?

### DISCUSSION THREAD

**Discussion by snowpeak1004** [fusiondevkr/pl900#57] [1 Q&A]
안녕하세요.  교육 완료자입니다. 
블로그에 간단히 후기 작성했는데, 어디에 등록(?)해야 하나요? 
https://yeripapa.tistory.com/13

**Comment by justinyoo** [comment]
감사합니다! 여기에 올려주시면 저희가 참고하겠습니다!

**Comment by sesme100** [comment]
후기 확인하였습니다. 자세하게 정리 해주셔서 감사합니다!!
